presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloErrorCode.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.accumulo;
+
+import com.facebook.presto.spi.ErrorCode;
+import com.facebook.presto.spi.ErrorCodeSupplier;
+import com.facebook.presto.spi.ErrorType;
+
+import static com.facebook.presto.spi.ErrorType.EXTERNAL;
+import static com.facebook.presto.spi.ErrorType.USER_ERROR;
+
+public enum AccumuloErrorCode
```

Comment:
I originally was using all `StandardErrorCodes`, but migrated them all over to a new `AccumuloErrorCode` class per [electrum's comment](https://github.com/prestodb/presto/pull/5030#issuecomment-211451292).  I took that to mean that all connector errors should have their own subclass, but I can use `StandardErrorCode` when an error code is available and use `AccumuloErrorCode` for Accumulo-based errors.